pkgs/applications/audio/mixxx/default.nix

```
@@ -118,9 +121,9 @@ mkDerivation rec {
     wavpack
   ];
 
-  qtWrapperArgs = [
-    "--set LOCALE_ARCHIVE ${glibcLocales}/lib/locale/locale-archive"
-  ];
+  preFixup=''
+    qtWrapperArgs+=(--set LOCALE_ARCHIVE $glibcLocales/lib/locale/locale-archive ''${gappsWrapperArgs[@]})
```

```suggestion
    qtWrapperArgs+=(--set LOCALE_ARCHIVE ${glibcLocales}/lib/locale/locale-archive ''${gappsWrapperArgs[@]})
```

It's a package, not a variable